hw/ip/otbn/README.md

```diff
@@ -66,6 +66,19 @@ DMem with `./example_dir/prog_bin_dmem.elf`. The `-t` argument enables tracing.
 The simulation automatically halts on an `ecall` instruction and prints the
 final register values.
 
+### Run the smoke test
+
+A smoke test which exercises some functionality of OTBN and the expected outputs
+for the smoke test (in the form of final register values) can be found in
```

Comment:
Nit: This sounds like the smoke test exercises its expected outputs. Maybe:
```
A smoke test which exercises some functionality of OTBN can be found, together with
its expected outputs, in `hw/ip/otbn/dv/smoke`
```

Or maybe
```
OTBN has a directed smoke test. This runs some fixed code and dumps the final register values,
checking them against known expected values. The test and the expected values can be found
in ...
```